front/app/components/admin/PostManager/components/PostTable/Row/selectors/TopicsSelector.tsx

```diff
@@ -1,47 +1,76 @@
-import React, { memo, useCallback, FormEvent } from 'react';
+import React, { memo, useCallback } from 'react';
 
+import {
+  IconButton,
+  colors,
+  Box,
+  stylingConsts,
+  Text,
+} from '@citizenlab/cl2-component-library';
 import { pull } from 'lodash-es';
-import { Label, Icon } from 'semantic-ui-react';
-import styled from 'styled-components';
 
 import useTopics from 'api/topics/useTopics';
 
 import T from 'components/T';
 
-import { isNilOrError } from 'utils/helperUtils';
+import { useIntl } from 'utils/cl-intl';
 
-const StyledLabel = styled(Label)`
-  white-space: nowrap;
-`;
+import messages from '../messages';
 
 interface Props {
   selectedTopics: string[];
   onUpdateTopics: (topicIds: string[]) => void;
 }
 
 const TopicsSelector = memo<Props>(({ selectedTopics, onUpdateTopics }) => {
+  const { formatMessage } = useIntl();
   const { data: topics } = useTopics();
   const filteredTopics = topics?.data.filter((topic) =>
     selectedTopics.includes(topic.id)
   );
   const handleTopicDelete = useCallback(
-    (topicId: string) => (event: FormEvent) => {
-      event.stopPropagation();
+    (topicId: string) => {
       const newSelectedTopics = pull(selectedTopics, topicId);
       onUpdateTopics(newSelectedTopics);
     },
     [selectedTopics, onUpdateTopics]
   );
 
-  if (!isNilOrError(topics)) {
+  if (topics) {
     return (
       <>
```

Comment:
Guess this needs to be a flexbox because they topics are now stacking vertically for me:

<img width="685" alt="image" src="https://github.com/user-attachments/assets/9f9f4221-0fc6-46b0-a7ec-3f1dc1abd694" />

Reply:
Nice catch!